CONTRIBUTORS.md

```
@@ -0,0 +1,274 @@
+# Jellyfin Contributors
+
+ - [1337joe](https://github.com/1337joe)
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 0; Actual: 1

(MD007, ul-indent)